Gini-iOS-SDK/GINIDocumentTaskManager.m

```diff
@@ -67,16 +67,35 @@ - (BFTask *)createDocumentWithFilename:(NSString *)fileName
                              fromImage:(UIImage *)image {
     return [self createDocumentWithFilename:fileName
                                   fromImage:image
-                                    docType:nil];
+                                   metadata:nil];
+}
+
+- (BFTask *)createDocumentWithFilename:(NSString *)fileName
+                             fromImage:(UIImage *)image
+                              metadata:(GINIDocumentMetadata *)metadata {
+    return [self createDocumentWithFilename:fileName
+                                   fromImage:image
+                                    docType:@"image/jpeg"
```

Comment:
How do you know it's a jpeg?

Reply:
I was also wondering with this is set as `image/jpeg`, but if you follow that method you see that it's later compressed as a JPEG. Since it's already deprecated, I preferred to keep it as it is right now.